cogs/remind.py

```diff
@@ -139,7 +139,7 @@ async def on_reminder_complete(self, record):
 
 		try:
 			if channel is not None:
-				await channel.send(content=f'<@{user_id}>', embed=e)
+				await channel.send(content=f'<@{user_id}>', embed=e, allowed_mentions=discord.AllowedMentions(users=True))
```

Comment:
Can probably be replaced with `allowed_mentions=discord.AllowedMentions(users=[discord.Object(user_id)])`